biomaj_download/download/curl.py

```diff
@@ -199,6 +215,10 @@ def _network_configuration(self):
         if self.credentials is not None:
             self.crl.setopt(pycurl.USERPWD, self.credentials)
 
+        # Hosts file & function to decide for new hosts
+        self.crl.setopt(pycurl.SSH_KNOWNHOSTS, self.ssh_hosts_file)
```

Comment:
Is it accapted on all curl version?
Else should be in a try/except, or set only if defined in config

Reply:
This is accepted since [7.19.6](https://curl.haxx.se/changes.html#7_19_6) which dates back to 2009 so I think it's OK.